api/src/main/java/de/valtech/aecu/api/service/AecuService.java

```diff
@@ -44,10 +44,22 @@
 public interface AecuService {
 
     /**
-     * Prefix in CRX where AECU scripts are located.
+     * Prefix in repository where AECU scripts are located.
+     * @deprecated Use either {@link #AECU_VAR_PATH_PREFIX} or {@link #AECU_CONTENT_PATH_PREFIX}
      */
-    public static final String AECU_PREFIX = "/var/groovyconsole/scripts/aecu";
+    @Deprecated
+    public static final String AECU_PREFIX = "/etc/groovyconsole/scripts/aecu";
```

Comment:
Why should we add the old location as deprecated? It is also not used in the code change to check for valid scripts.

Reply:
Only the old constant name is deprecated, the old location is still valid!

Reply:
But the value must not change of course, gonna fix it.